### ISSUE TITLE
Give authors a way to suppress right-hand table of content entries

### ISSUE THREAD

**Issue by nschonni** [MicrosoftDocs/feedback#1194]
**Is your feature request related to a problem? Please describe.**
I've come across this in a few repos:
- https://github.com/MicrosoftDocs/VBA-Docs/issues/800
-https://github.com/MicrosoftDocs/visualstudio-docs/pull/2741
Authors are skipping heading levels in order to not have an entry in the generated ToC. This causes HTML semantic issues, and in turn issues for screen reader users.

**Describe the solution you'd like**
Give a way to suppress the generation of ToC entries for an article through inline markup, or page level metadata.

**Describe alternatives you've considered**
The H2s shouldn't be suppressed and should be shown, even if the author finds them to be low-value

**Additional context**
This can usually be found in repos when running MarkdownLint rule MD001 https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md001

**Comment by rootsmusic** [comment]
@nschonni Is this issue about authoring with VS Code's extension at https://marketplace.visualstudio.com/items?itemName=docsmsft.docs-markdown ?

**Comment by nschonni** [comment]
No. It could be something that would be exposed there, but the issue is with the build engine/template that auto-promotes all H2s to the side nav
